src/main_app/tasks/uploads/upload_bot.py

```
@@ -8,11 +8,12 @@
 
 
 def upload_file(
-    file_name,
-    file_path,
-    site=None,
-    summary=None,
-    new_file=False,
+    file_name: str,
+    file_path: Path,
+    site: mwclient.client.Site = None,
+    summary: str = None,
```

Ruff (0.15.2)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)